services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/allu/CableReportServiceAllu.kt

```diff
@@ -296,6 +298,30 @@ class CableReportServiceAllu(
             .orElseThrow()
     }
 
+    /** Send a comment to the application with Haitaton system as the sender. */
```

Comment:
The comment could explain the returned Int.

Reply:
Added an explanation of the return value.